airflow/hooks/druid_hook.py

```diff
@@ -83,6 +84,9 @@ def construct_ingest_query(
         else:
             num_shards = -1
 
+        query_granularity = query_granularity or "NONE"
+        segment_granularity = segment_granularity or "DAY"
```

Comment:
that makes sense, didn't consider people using the function directly, fixed that